### DISCUSSION TITLE
Looking for a module to simulate fracture propagation

### DISCUSSION THREAD

**Discussion by miraboreasumr** [idaholab/moose#23096] [Q&A General] [answered]
Hello, sorry for posting a similar question. Here is what I need for the module
1. I have coordinates for the Naturally Fractures which I need to input
2. I inject water into the fractures, then it propagates (so maybe no need for flow in porous media)
3. Maybe no need for poroelastic, just elastic.
Thank you

**Comment by hugary1995** [comment, marked as answer]
There is no single module for hydraulic fracture in porous media. You can develop your own application leveraging one or multiple existing modules: porous flow, tensor mechanics and possibly phase field/XFEM.
